tests/vm-prep

```diff
@@ -64,6 +64,12 @@ prepare()
 		$VIRSH net-undefine $NETWORK_NAME
 	fi
 
+    # In case a network has leaked in from elsewhere
+    if ip address show dev $NETWORK_NAME >/dev/null 2>/dev/null; then
+        ip link set $NETWORK_NAME down || true
+        brctl delbr $NETWORK_NAME || true
+    fi
+
```

Comment:
> The indentation is off.

Various other places in that file are moving to 4 space indentantion. So I figured I would follow suit here.

> We remove it above. It's because a Pod restarts containers inside of it, libvirt is running inside of the containers, but the network namespace is owned by the Pod. And therefore the network outlasts libvirt and the container.